PaymentStatementIntegrations/environments/dev/settings-dev.json

```diff
@@ -41,22 +41,22 @@
   },
   {
     "name": "PortalClientId",
-    "value": "@Microsoft.KeyVault(SecretUri=https://devrleinfkv1401.vault.azure.net/secrets/PortalClientId/8241de3ad8d64f3282a9346c2ab5f0a8)",
+    "value": "@Microsoft.KeyVault(VaultName=devrleinfkv1401;SecretName=PortalClientId)",
```

Comment:
You 'can' use KV refs of KVname/SecretName - but you can hit issues with caching where you need to restart your app service for it to pick up a new value (if the KV secret has been changed since last restart). However, using explicit guid refs avoids any caching issues and you definitely know whch secret it will pick up

Reply:
Replacing with secret name reference.